testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -567,6 +567,18 @@ impl BitcoinRegtestController {
         result_vec
     }
 
+    /// Checks if there is a default wallet with the name of "".
+    /// If the default wallet does not exist, this function creates a wallet with name "".
+    pub fn create_wallet_if_dne(&self) {
+        let wallets =
+            BitcoinRPCRequest::list_wallets(&self.config).expect("Call to listwallets failed");
```

Comment:
This code could panic if the bitcoin node was slow, or if your device was having connectivity issues.  Could you instead have this method return an error if the RPC fails, so the caller can report an appropriate error message?

Reply:
(Same goes for the call to `create_wallet`.  Generally speaking, networking code that could fail should not panic).

Reply:
Ok, now handling the error with a warn statement.

Reply:
Added a call to `warn!` instead of panicking now.